client/dist/leaderboard.css

```diff
@@ -0,0 +1,7 @@
+.leaderboard-container {
+  overflow: ;
```

Comment:
never seen a container insecure about how it should overflow 🫣

Reply:
ill fix it later :)